src/main/java/com/segment/analytics/android/integrations/nielsendcr/NielsenDCRIntegration.java

```diff
@@ -166,11 +237,26 @@ private JSONObject mapSpecialKeys(
       contentMetadata.put("segC", segC);
     }
 
+    if (options.containsKey("hasAds") && "true".equals(String.valueOf(options.get("hasAds")))) {
```

Comment:
I think `options.get("hasAds")` can be `null` which would result in a null pointer exception when `String.valueOf` is applied to it.